x-pack/plugins/security/server/routes/users/disable.ts

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { schema } from '@kbn/config-schema';
+import { RouteDefinitionParams } from '../index';
+import { wrapIntoCustomErrorResponse } from '../../errors';
+import { createLicensedRouteHandler } from '../licensed_route_handler';
+
+export function defineDisableUserRoutes({ router }: RouteDefinitionParams) {
+  router.post(
+    {
```

Comment:
We should add API tests for these new routes. Unit-level tests probably aren't necessary since these are so small.

We have other security API tests located in `x-pack/test/api_integration/apis/security`, so this would be a good place to add some for these endpoints.

Reply:
Nice, I'll add those in